src/mdb_v8.c

```diff
@@ -3362,6 +3444,60 @@ jsobj_print_jsarray(uintptr_t addr, jsobj_print_t *jsop)
 }
 
 static int
+jsobj_print_jstyped_array(uintptr_t addr, jsobj_print_t *jsop)
```

Comment:
That seems reasonable, but so does doing nothing special (as long as there's another way to get the array data out).

Arrays can have named properties, too, and we don't print those.  We could, but then we couldn't print an Array the way it would look inline (since there's no place for other named properties), which is more convenient most of the time.  We could probably do better here more broadly, but I don't think it's been a major pain point.